ChangeLog.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v0.3.9] - 2024-08-15
```

Comment:
The version stamping should be done at release time https://github.com/Learnosity/learnosity-sdk-python/blob/master/CONTRIBUTING.md#deploying-to-pypi

The ChangeLog will be automatically updated to include all `Unreleased` changes.

Reply:
Thanks @shtrom for the advice! I'll revert those changes now 👍

Reply:
I've updated the changelog now and added this under the `Unreleased` section.

@shtrom could you check it conforms with the usual standards?